simulation/lowfi_simulation/src/main/native/cpp/lowfisim/SimulatorComponentBase.cpp

```diff
@@ -0,0 +1,25 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "lowfisim/SimulatorComponentBase.h"
+
+namespace frc {
+namespace sim {
+namespace lowfi {
+SimulatorComponentBase::SimulatorComponentBase() {}
```

Comment:
Use = default in the header instead of placing an empty constructor and destructor here. If only default constructors are specified, they can be omitted entirely.